examples/07-Train-traditional-ML-models-using-the-Merlin-Models-API.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Suggestion: output the head of valid before the predict call and describe what is being passed (user ids). The prediction output is a list of item ids, per each user, to be interacted with, ordered from most likely to least likely (indicated by the values in the second array).</p><p> </p><p>Also, what is the meaning or definition of the numbers 4.0, 3.9, 3.89, etc.? They are not probabilities, so what are they? Are they scaled out of 100, etc.? A bit more explanation here so the user can make a better sense of these numbers. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>added text addressing the points you raise</p>

Reply:
<p>Thanks for the explanations on ids, scores, etc.; it reads well. I noticed a double "the" in the second last paragraph, please remove one.</p><p> </p>